src/services/oathkeeper/index.ts

```diff
@@ -0,0 +1,49 @@
+import { decisionsApi } from "@config"
+import {
+  OathkeeperUnauthorizedServiceError,
+  OathkeeperError,
+  OathkeeperForbiddenServiceError,
+  UnknownOathkeeperServiceError,
+  OathkeeperMissingAuthorizationHeaderError,
+} from "@domain/oathkeeper/errors"
+import axios from "axios"
+
+export const sendOathkeeperRequest = async (
```

Comment:
I thought the call graph is oath keeper -> galoy not vica versa. This introduces a circular dependency...